crates/aptos/CHANGELOG.md

```diff
@@ -1,92 +1,52 @@
-# Aptos CLI Changelog
+# Movement CLI Changelog
 
-All notable changes to the Aptos CLI will be captured in this file. This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html) and the format set out by [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+All notable changes to the Movement CLI will be captured in this file. This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html) and the format set out by [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
-## Unreleased
-- Add balance command to easily get account balances for APT currently
-
-## [3.4.1] - 2024/05/31
```

Comment:
This is a revert of the entire `crates/aptos` to the old revision that was pulled from upstream in #20, is it?

Reply:
Basically. Do you want me to try and cherry-pick it?